xml-java8/shared/src/main/scala/rapture/xml/java8/time.scala

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
This looks very similar to the JSON version... could we just have a single implementation which doesn't need the `Ast` implicits?